pipelines/internal/commands/run/run.go

```diff
@@ -155,7 +154,8 @@ var (
 	cloudSDKImage  = flags.String("cloud-sdk-image", "google/cloud-sdk:alpine", "the cloud SDK image to use")
 	timeout        = flags.Duration("timeout", 0, "how long to wait before the operation is abandoned")
 	defaultImage   = flags.String("image", "bash", "the default image to use when executing commands")
-	attempts       = flags.Uint("attempts", 1, "number of attempts on non-fatal failure")
+	attempts       = flags.Uint("attempts", 0, "number of attempts on non-fatal failure, using non-preemptible VM")
+	pmbAttempts    = flags.Uint("preemptible-attempts", 1, "number of attempts on non-fatal failure, using preemptible VM")
```

Comment:
Let's have the flag just be pvm-attempts (easier to type) and the variable should be pvmAttempts